ignore.bii

```diff
@@ -0,0 +1,9 @@
+build/**
```

Comment:
This file excludes certain directories from being published to the biicode repository